Wangscape/NoiseModuleManager.cpp

```diff
@@ -0,0 +1,53 @@
+#include "NoiseModuleManager.h"
+#include "ModuleFactories.h"
+#include <random>
+#include <time.h>
+
+NoiseModuleManager::NoiseModuleManager(const Options & options) :
+    mRNG((unsigned int)time(nullptr))
+{
+    for (const auto& terrain : options.terrains)
+    {
+        mStochasticMasks.insert({terrain.first, makePlaceholder().scaleBias(0.5, 0.5)});
```

Comment:
This loop uses the keys from `options.terrains`, but `std::transform` on a `std::map` seems to give access only to the values. I don't think `std::transform` can be used here unless the structure of `options.terrains` changes.